ext/cdi/jersey-cdi1x-servlet/src/main/java/org/glassfish/jersey/ext/cdi1x/servlet/internal/CdiExternalRequestScopeExtension.java

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2015, 2018 Oracle and/or its affiliates. All rights reserved.
+ * (Portions) Copyright (c) 2018 Payara Foundation and/or its affiliates.
```

Comment:
I am sorry I rerun with the copyright plugin and Portions copyright does not work. Only simple "Copyright works".

Reply:
I can change it if you want

Reply:
Should I remove the "(Portions)" from copyright header ?

Reply:
Yes please. I'm sorry again. Should have tested it earlier.

Reply:
It's okay @jansupol, I have updated the header and thanks for the review.